code/game/objects/items/toys.dm

```diff
@@ -503,207 +503,324 @@
+	toysay = "Standard Operating Procedure says they're guilty! Hacking is proof they're an Enemy of the Corporation!"
 
 /obj/item/toy/figure/librarian
 	name = "Librarian action figure"
 	desc = "A \"Space Life\" brand Librarian action figure."
 	icon_state = "librarian"
+	toysay = "One day while..."
 
 /obj/item/toy/figure/md
 	name = "Medical Doctor action figure"
 	desc = "A \"Space Life\" brand Medical Doctor action figure."
 	icon_state = "md"
+	toysay = "The patient is already dead!"
 
 /obj/item/toy/figure/mime
 	name = "Mime action figure"
 	desc = "A \"Space Life\" brand Mime action figure."
 	icon_state = "mime"
+	toysay = "..."
 
 /obj/item/toy/figure/miner
 	name = "Shaft Miner action figure"
 	desc = "A \"Space Life\" brand Shaft Miner action figure."
 	icon_state = "miner"
+	toysay = "Oh god, it's eating my intestines!"
 
 /obj/item/toy/figure/ninja
 	name = "Space Ninja action figure"
 	desc = "A \"Space Life\" brand Space Ninja action figure."
 	icon_state = "ninja"
+	toysay = "Oh god! Stop shooting, I'm friendly!"
 
 /obj/item/toy/figure/wizard
 	name = "Wizard action figure"
 	desc = "A \"Space Life\" brand Wizard action figure."
 	icon_state = "wizard"
+	toysay = "Ei Nath!"
 
 /obj/item/toy/figure/rd
 	name = "Research Director action figure"
 	desc = "A \"Space Life\" brand Research Director action figure."
 	icon_state = "rd"
+	toysay = "Blowing all of the borgs!"
 
 /obj/item/toy/figure/roboticist
 	name = "Roboticist action figure"
 	desc = "A \"Space Life\" brand Roboticist action figure."
 	icon_state = "roboticist"
+	toysay = "He asked to be borged!"
 
 /obj/item/toy/figure/scientist
 	name = "Scientist action figure"
 	desc = "A \"Space Life\" brand Scientist action figure."
 	icon_state = "scientist"
+	toysay = "Someone else must have made those bombs!"
 
 /obj/item/toy/figure/syndie
 	name = "Doom Operative action figure"
 	desc = "A \"Space Life\" brand Doom Operative action figure."
 	icon_state = "syndie"
+	toysay = "Get that fucking disk!"
 
 /obj/item/toy/figure/secofficer
 	name = "Security Officer action figure"
 	desc = "A \"Space Life\" brand Security Officer action figure."
 	icon_state = "secofficer"
+	toysay = "I am the law!"
+
+/obj/item/toy/figure/virologist
+	name = "Virologist action figure"
+	desc = "A \"Space Life\" brand Virologist action figure."
+	icon_state = "virologist"
+	toysay = "The cure is potassium!"
 
 /obj/item/toy/figure/warden
 	name = "Warden action figure"
 	desc = "A \"Space Life\" brand Warden action figure."
 	icon_state = "warden"
+	toysay = "Execute him for breaking in!"
 
 /obj/item/toy/figure/psychologist
 	name = "Psychologist action figure"
 	desc = "A \"Space Life\" brand Psychologist action figure."
 	icon_state = "psychologist"
+	toysay = "The analyzer says you're fine!"
 
 /obj/item/toy/figure/paramedic
 	name = "Paramedic action figure"
 	desc = "A \"Space Life\" brand Paramedic action figure."
 	icon_state = "paramedic"
+	toysay = "WHERE ARE YOU??"

```

Comment:
stop using allcaps and multiple punctuation marks if our players arent meant to

Reply:
... they're toys, though? If your players say "I can use allcaps because this ingame toy does", ban them because you'd be better off without them.

Reply:
It just gives the impression of using it being proper; the standard in my past PRs has been to minimize it with use of italics and bold. That's part of why we have both available for speech as well; it's a minor thing and you can leave it anyway, I'm just personally opposed because it's a peeve to see "HELP ME!!!!!" on the radio.

Reply:
All of the toys MOCK the players/jobs. They are not giving an impression of something being proper, they are all jokes.

Reply:
That's more of an OOC thing, though, the all caps punctuation thing.